testdata/configtest.json

```diff
@@ -1,5 +1,6 @@
 {
```

Comment:
Please revert these changes as we use `configtest` to check config change compatibility